example_test.go

```diff
@@ -56,3 +59,51 @@ func Example() {
 	// All outgoing requests from client will include a Stackdriver Trace header.
 	// See the ochttp package for how to handle incoming requests.
 }
+
+func Example_gKE() {
```

Comment:
This is the only spelling of GKE I could find to get godoc to render it correctly as GKE.